app/resources/api/v1/job_resource.rb

```diff
@@ -49,6 +52,10 @@ class JobResource < JSONAPI::Resource
                   .all
              end)
 
+      def self.sortable_fields(context)
```

Comment:
Is this for the inbox only?/ maybe we should not include it in this particular pull request as there might be more to do

Reply:
This is the feature I was trying to implement with the PR (could have named it better). Hopefully the Vue Inbox app will be able to make GET requests and that sort on these values now. The Work Plan stuff doesn't serve much purpose currently, but does allow GETting of work plan info through a JSON API now.